packages/sentry/src/adapters/sentryMonitoringAdapter.ts

```diff
@@ -0,0 +1,161 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
- Rename this file to `adapter.ts`
- Move this file into the `src` directory
- Remove the `adapters` directory